cdc/processor/pipeline/sorter.go

```diff
@@ -94,6 +103,9 @@ func (n *sorterNode) Init(ctx pipeline.NodeContext) error {
 		return nil
 	})
 	n.wg.Go(func() error {
+		lastSentResolvedTs := uint64(0)
+		lastSendResolvedTsTime := time.Now()
```

Comment:
Is there any relation between `lastSentResolvedTs` and `lastSendResolvedTsTime`, why not  `lastSentResolvedTs := lastSendResolvedTsTime.Unix()`? should timezone be considered ?

Reply:
1. `lastSentResolvedTs` must be a valid TiDB timestamp, which is `(physicalTs << 18 + logicalTs)`.
2. `lastSendResolvedTsTime` is the time at which this pipeline staged has sent a Resolved Event. It is a clock time, not directly related to the TiDB timestamp.
3. There is no need to consider timezone, because 1) TiDB timestamps are timezone-less 2) Golang's `time.Time` takes care of timezone internally.